Rubberduck.Core/UI/UnitTesting/TestExplorerControl.xaml

```diff
@@ -216,6 +216,18 @@
 
                 <Separator />
 
+                <Label Content="{Resx Key=TestExplorer_Filter, ResxName=Rubberduck.Resources.UnitTesting.TestExplorer}" />
+                <controls:SearchBox Width="100"
+                                    Text="{Binding TestNameFilter, Mode=OneWayToSource, UpdateSourceTrigger=PropertyChanged}" />
+
+                    <Label Content="{Resx Key=TestExplorer_Outcome, ResxName=Rubberduck.Resources.UnitTesting.TestExplorer}" />
+
+                <ComboBox Width="100" 
+                          ItemsSource="{Binding OutcomeFilters, UpdateSourceTrigger=PropertyChanged}"
+                          SelectedItem="{Binding SelectedOutcomeFilter, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" />
```

Comment:
`TestNameFilter` is marked `OneWayToSource`. Why is this one not?

Reply:
Additionally I recall there being the option to filter for multiple outcomes using ToggleButtons similar to the grouping behaviour... Where'd that go?